server/data.js

```diff
@@ -0,0 +1,105 @@
+const _ = require('underscore');
+
+// 'YYYY-MM-DD HH:mm:ss' for MySQL insertion (extra quotes yes, 0 padding not needed)
+const format = (dateTime) => {
+  const y = dateTime.getFullYear();
+  const mon = dateTime.getMonth() + 1;
+  const d = dateTime.getDate();
+  const h = dateTime.getHours();
+  const min = dateTime.getMinutes();
+  const s = dateTime.getSeconds();
+  return `"${y}-${mon}-${d} ${h}:${min}:${s}"`;
+};
+
+/*
+ *  Generate data for the "inventory" table
+ */
+
+const partyTypes = {
+  // all: _.range(1, 21),
+  small: _.range(1, 3),
+  large: _.range(19, 21),
+  scattered: [1, 10],
+};
+
+// 2019-12-01 00:00:00 to 2020-03-31 23:30:00 in 30-min increments
+// values in milliseconds from 1970-01-01 00:00:00
```

Comment:
nice descriptive annotations like this for bigger block of functions are very helpful for code review and for your future self when you need to revisit the code.

Reply:
noted